backend/settings.py

```diff
@@ -210,9 +210,10 @@
 
 TINYMCE_DEFAULT_CONFIG = {
         'menubar': True,
-        "plugins": "advlist,autolink,lists,link,anchor,insertdatetime,media,table,paste,",
+        "plugins": "advlist,autolink,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
```

Comment:
Is there a `charcount`? Seems like that would be more useful in the `short_description` case, but maybe words are better in other fields.

Reply:
If you click the word count in the status bar it changes to character count. I didn't see any option to switch the defaults other than to use javascript which the django-tinymce plugin doesn't support yet as part of the init and it looks clunky.

They wanted me to do contribute a patch to support this to also allow for dark mode but I haven't gotten that yet. You're right I'm also not sure how it would interact with this custom validator and if it would return the same HTML-less count. I can also check that and remove it if it doesn't.

Reply:
Yeah this count looks consistent with what's displayed.